cmd/version.go

```diff
@@ -25,38 +24,6 @@ func (cli *CLI) printVersion(opts Options) int {
 	}
 	cfg = cfg.Merge(opts.toConfig())
 
-	cli.loader, err = tflint.NewLoader(afero.Afero{Fs: afero.NewOsFs()}, cfg)
-	if err != nil {
-		log.Printf("[ERROR] Failed to prepare loading: %s", err)
-		return ExitCodeOK
-	}
-
-	runners, appErr := cli.setupRunners(opts, cfg, ".")
-	if appErr != nil {
-		log.Printf("[ERROR] Failed to setup runners: %s", appErr)
-		return ExitCodeOK
-	}
-	rootRunner := runners[len(runners)-1]
-
-	// AWS plugin is automatically enabled from your provider requirements, even if the plugin isn't explicitly enabled.
-	if _, exists := cfg.Plugins["aws"]; !exists {
-		reqs, diags := rootRunner.TFConfig.ProviderRequirements()
-		if diags.HasErrors() {
-			log.Printf("[ERROR] Failed to get Terraform provider requirements: %s", diags)
-			return ExitCodeOK
-		}
-		for addr := range reqs {
-			if addr.Type == "aws" {
-				log.Print("[INFO] AWS provider requirements found. Enable the plugin `aws` automatically")
-				fmt.Fprintln(cli.errStream, "WARNING: The plugin `aws` is not explicitly enabled. The bundled plugin will be enabled instead, but it is deprecated and will be removed in a future version. Please see https://github.com/terraform-linters/tflint/pull/1160 for details.")
```

Comment:
Thoughts on keeping a warning with new wording?

Reply:
That's a good question. I thought about that, but I stopped displaying warnings because it would be an obvious noise for users who are only using Terraform rules.